cleanup-db-python.py

```
@@ -0,0 +1,133 @@
+import subprocess
+import sys
+import os
+
+print("=" * 60)
+print("Git Users Database Cleanup - Direct SQL Method")
+print("=" * 60)
+print()
+
+# Step 1: Stop Ignition
+print("Step 1: Stopping Ignition container...")
+result = subprocess.run(["docker", "stop", "whk-services-ignition-1"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)